ext/lib/crypto/mbedtls/Kconfig

```diff
@@ -17,6 +17,14 @@
 #
 
 
+config MBEDTLS_EXTERNAL_CONFIG
+	bool
+	help
+	  This internal option controls if mbedTLS is configured by external
+	  Kconfig file.
```

Comment:
If this is needed, maybe it could be called something like `HAS_CUSTOM_MBEDTLS_KCONFIG`. That might be a bit more idiomatic, especially if it is `select`ed.

Reply:
I can change the name, but I don't think that `HAS_CUSTOM_MBEDTLS_KCONFIG` is right name. All other mbedTLS related entries have `MBEDTLS_` at the beginning, so I think that changing that approach is not good. What about `MBEDTLS_HAS_CUSTOM_KCONFIG`?

Reply:
Regarding your previous question. Yes, there are different versions of mbedTLS. Mostly they all share the same API, but may have different backends (e.g. HW accelerated). Because of that creating one common configuration in this place is not possible.

Reply:
It is not clear to me why the mbedtls in Zephyr shouldn't be extended to support a HW accelerated backend.

Generally this is less complicated than integrating out-of-tree Kconfig+CMake code.

Reply:
It depends how deep changes are required to support such backed. If there are a lot of changes then different fork of mbedTLS with some Zephyr specific patches will be required. Maintaining this fork to be up to date with official mbedTLS fork would take a lot of effort. Additionally there may be situation when some licence restrictions prevent adding some sources to open source project which is Zephyr.